### DISCUSSION TITLE
Processsing's Table data structure

### DISCUSSION THREAD

**Discussion by villares** [py5coding/py5generator#477] [Show and tell]
![image](https://github.com/py5coding/py5generator/assets/3694604/d858da99-8bfc-4cb6-8ea8-bdb4d99ee599)

I guess I could use a CSVDictWriter or something, to export CSVs, but I'm porting Processing Python mode code from 3 or 4 years ago, a tool I made for my colleagues at my supervisor's research group, and I'd rather just have it working before I refactor it into something more elegant (not very confident I will, really).

On Processing 3.5.4 with Python mode I used `from processing.data import Table`.
Processing 4 docs don't mention any imports: https://processing.org/reference/Table.html
I panicked...
...but it worked the same as on 3.5.4 on py5, yay!

![image](https://github.com/py5coding/py5generator/assets/3694604/c32aa13e-b8e9-4c5e-afc9-c1772ed7482a)

**Comment by hx2A** [comment]
Interesting. In that code, `Table` is a Java object, not a Python object. I intentionally skipped over Processing's Table class because I figured nobody would want to use it if they could use pandas instead.  Although a pandas DataFrame has a lot of moving parts and might therefore be intimidating for a beginner coder.

**Reply by villares** [reply]
I hope jpype with handle my arguments well when calling its methods, I'll keep that information on mind.
